contactsummary/js/src/contact-summary/services/api.js

```diff
@@ -45,7 +45,6 @@ define([
          */
         function sendRequest(method, data, config) {
             config = angular.extend({
-                cache: true,
```

Comment:
Can you document this change in the PR description? Seems important to me in terms of reasons and consequences